packages/credential-provider-node/package.json

```diff
@@ -36,7 +36,6 @@
     "tslib": "^1.8.0"
   },
   "devDependencies": {
-    "@aws-sdk/shared-ini-file-loader": "3.4.1",
```

Comment:
Verified that `@aws-sdk/shared-ini-file-loader` is added as a devDependency on line 34